spot1:
  odom_topic: "/spot1/base/spot/odometry"
  depth_left: "/spot1/base/spot/depth/frontleft/image"
  depth_right: "/spot1/base/spot/depth/frontright/image"
  rgb_left: "/spot1/base/spot/camera/frontleft/image"
  rgb_right: "/spot1/base/spot/camera/frontright/image"
  accum_topic: "/spot1/base/spot/depth/accum_front"
  occup_topic: "/spot1/base/spot/depth/occup_front"
  normal_topic: "/spot1/base/spot/depth/normal_front"
  combined_image_raw: "/spot1/base/spot/camera/combined_image_raw"
  combined_image_similarity: "/spot1/base/spot/camera/combined_image_similarity"
  body_frame: "spot1/base/spot/body"
  odom_frame: "spot1/base/spot/odom"

spot2:
  odom_topic: "/spot2/base/spot/odometry"
  depth_left: "/spot2/base/spot/depth/frontleft/image"
  depth_right: "/spot2/base/spot/depth/frontright/image"
  rgb_left: "/spot2/base/spot/camera/frontleft/image"
  rgb_right: "/spot2/base/spot/camera/frontright/image"
  accum_topic: "/spot2/base/spot/depth/accum_front"
  occup_topic: "/spot2/base/spot/depth/occup_front"
  normal_topic: "/spot2/base/spot/depth/normal_front"
  combined_image_raw: "/spot2/base/spot/camera/combined_image_raw"
  combined_image_similarity: "/spot2/base/spot/camera/combined_image_similarity"
  body_frame: "spot2/base/spot/body"
  odom_frame: "spot2/base/spot/odom"